spec/views/documents/show.html.erb_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+RSpec.describe "documents/show.html.erb" do
+  include TopicsHelper
+  before { populate_default_government_bulk_data }
+
+  describe "document summary" do
+    it "shows the document and edition metadata" do
+      edition = create(:edition,
+                       title: "Title",
+                       summary: "Summary",
+                       last_edited_by: create(:user, name: "User 1"),
+                       created_by: create(:user, name: "User 2"))
+      assign(:edition, edition)
+      render
+
+      expect(rendered).to include("Title")
+      expect(rendered).to include("Summary")
+      expect(rendered).to have_selector(
+        ".app-c-metadata",
+        text: /#{I18n.t!("documents.show.metadata.created_by")}:\s*User 2/,
+      )
+      expect(rendered).to have_selector(
+        ".app-c-metadata",
```

Comment:
Minor: do we need to use a selector approach? As long as the text is specific enough, I generally prefer to just check for that, unless it's clearly ambiguous.

Reply:
Good point - selector wasn't a preferred approach just a result of me getting more specific to get a match. Of course that was before I switched to a regex so that was just cruft left from a previous approach.